draft-02.txt

```diff
@@ -1315,6 +1321,12 @@ Internet-Draft     Application-Level Profile Semantics        March 2015
 
    Interoperability considerations:  None
 
+   Fragment identifier considerations:  Fragment identifiers used with
+      application/alps+json resources are simple, opaque strings that
```

Comment:
Yes. Feel free to commit that.
On Apr 14, 2015 20:11, "Mark W. Foster" <notifications@github.com> wrote:

> In draft-02.txt
> <https://github.com/alps-io/spec/pull/72#discussion_r28390660>:
>
> > @@ -1315,6 +1321,12 @@ Internet-Draft     Application-Level Profile Semantics        March 2015
> >
> >     Interoperability considerations:  None
> >
> > +   Fragment identifier considerations:  Fragment identifiers used with
> > +      application/alps+json resources are simple, opaque strings that
>
> @mamund <https://github.com/mamund> How about:
>
> opaque strings that do not contain any URL unsafe characters per
> http://www.rfc-base.org/txt/rfc-1738.txt which ...
>
> I think we should then add some similar wording to the id definition.
> Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alps-io/spec/pull/72/files#r28390660>.
>